skyflow/service_account/_validity.py

```diff
@@ -12,6 +12,9 @@ def is_expired(token: str):
     '''
     Check if stored token is not expired, if not return a new token, 
     if the token has expiry time before 5min of current time, call returns False
+
+    :param token: The token to be checked for expiration.
+    :returns: Returns `True`, if the token is expired or invalid, `False` otherwise.
```

Comment:
If the token is expired or will expire within 5 minutes, returns `True`. Otherwise, returns `False`.